code/game/gamemodes/hivemind/hivemind.dm

```diff
@@ -1,3 +1,6 @@
+/datum/game_mode
+	var/datum/objective/hivemind/assimilate_common/assimilation_obj //Create the versus objective here since we want a common target for all the antags
```

Comment:
why does every game mode require this var?

Reply:
Same reason why traitors have a couple of vars in /datum/game_mode to handle the exchange objectives

Reply:
reeeeee